content/en/docs/installing-on-linux/introduction/intro.md

```diff
@@ -13,15 +13,15 @@ As an open-source project on [GitHub](https://github.com/kubesphere), KubeSphere
 
 Users are provided with multiple installation options. Please note not all options are mutually exclusive. For instance, you can deploy KubeSphere with minimal packages on multiple nodes in an air-gapped environment.
 
-- [All-in-One](../all-in-one): Install KubeSphere on a single node. It is only for users to quickly get familiar with KubeSphere.
-- [Multi-Node](../multi-node): Install KubeSphere on multiple nodes. It is for testing or development.
-- [Install KubeSphere on Air-gapped Linux](../install-ks-on-linux-airgapped): All images of KubeSphere have been encapsulated into a package. It is convenient for air-gapped installation on Linux machines.
-- [High Availability Installation](../master-ha): Install high availability KubeSphere on multiple nodes which is used for the production environment.
+- [All-in-One](../../../quick-start/all-in-one-on-linux/): Install KubeSphere on a single node. It is only for users to quickly get familiar with KubeSphere.
+- [Multi-Node](../multioverview/): Install KubeSphere on multiple nodes. It is for testing or development.
+- [Air-gapped Installation on Linux](../air-gapped-installation): All images of KubeSphere have been encapsulated into a package. It is convenient for air-gapped installation on Linux machines.
+- High Availability Installation: Install high availability KubeSphere on multiple nodes which is used for the production environment.
 - Minimal Packages: Only install the minimum required system components of KubeSphere. Here is the minimum resource requirement:
   - 2vCPUs
   - 4GB RAM
   - 40GB Storage
-- [Full Packages](../complete-installation): Install all available system components of KubeSphere such as DevOps, service mesh, and alerting.
```

Comment:
Do we have this guide?

Reply:
I assume you are going to add **Enable Pluggable Components** here.

Reply:
No, we do not have the guide, which is why I deleted this link. I will change it to Enable Pluggable Components then.